backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -48,14 +50,13 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
       CameraController cameraController = ServerController.getInstance().getCameraController();
       Camera camera = cameraController.getCameraById(Integer.parseInt(camID));
       String tags = request.getParameter("tags");
-      List<String> tagList = null;
-      
+
+      List<String> tagList = new ArrayList<String>();
```

Comment:
If I remove <String>  then it gives a warning: ArrayList is a raw type. References to generic type ArrayList<E> should be parameterized. Do you think it is better to just say List = new arraylist and get a warning or use <String> ?